pkg/types/attributes.go

```diff
@@ -99,6 +93,16 @@ const (
 	AttrCaption = "caption"
 	// AttrStyle block or list style
 	AttrStyle = "style"
+	// AttrWidth the `width` attribute used ior images, tables, and so forth
+	AttrWidth = "width"
```

Comment:
Maybe we should also rename `AttrImageHeight` to `AttrHeight` for the sake of consistency?